build.sbt

```diff
@@ -0,0 +1,28 @@
+val appName = "address-lookup-performance-tests"
+val appVersion = "0.1.0-SNAPSHOT"
```

Comment:
Is there a reason we are using SNAPSHOT in the version here instead of just 0.1.0?